extensions-core/s3-extensions/src/main/java/org/apache/druid/storage/s3/output/S3StorageConnector.java

```diff
@@ -329,11 +335,14 @@ private void deleteKeys(List<DeleteObjectsRequest.KeyVersion> versions) throws I
   public void deleteRecursively(String dirName) throws IOException
   {
     try {
+      final String fullPath = objectPath(dirName);
+      log.debug("Deleting directory at bucket: %s, path: %s", config.getBucket(), fullPath);
+
```

Comment:
Please push these logging line to the s3Utils class so that other callers also benefit from it.

Reply:
Done